stdlib/Dates/src/types.jl

```diff
@@ -350,6 +350,7 @@ function ==(a::Time, b::Time)
         microsecond(a) == microsecond(b) && nanosecond(a) == nanosecond(b)
 end
 (==)(x::TimeType, y::TimeType) = (===)(promote(x, y)...)
+Base.hash(x::Time, h::UInt) = hash(hour(x), hash(second(x), hash(second(x), h)))
```

Comment:
Did you mean to include `minute` instead of `second` twice?

Reply:
Yeah, I was having source build problems, so I didn't get a chance to try any of this out locally. I think I've got it sorted out now.